nhss.c

```diff
@@ -7,6 +7,9 @@
 #include "move.h"
 #include "record.h"
 
+extern int record_add(const char);
```

Comment:
I put these two lines in `record.h` instead.

Reply:
Hmm, that might make more sense. Not sure
Yup! Works too right? - I got my BoH 🥳

Reply:
Congrats! I save-scummed the second level in a moment of weakness (haven't played in a year, so a bit rusty) before rediscovering this tool. I promise I'll be good now.

Reply:
I figure the `record.h` header is the public interface to `record.c` and that declares all the functions called by the main program. That way you also don't need `extern` declarations, just plain ones in the header.

Reply:
Ah I see, fixed!
Btw it seems to be missing a level or two (in 3.6.7 at least) but works fine to copy/paste from your game/wiki just don't get fooled by mimics!